lib/internal/Magento/Framework/View/Layout/CacheKeyInterface.php

```diff
@@ -0,0 +1,26 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\Framework\View\Layout;
+
+/**
+ * Interface CacheKeyInterface
+ */
+interface CacheKeyInterface
+{
+    /**
+     * Add cache key for generating different cache id for same handles
+     *
+     * @param array|string $cacheKey
```

Comment:
This method may accept array, to be consistent with `getCacheKeys`

Reply:
This method already accepts array as parameters as specified in @param notation, do you mean methods names are not consistent between them? (addCacheKey vs. getCacheKeys)

Or maybe addCacheKey should be called addCacheKeys if it accepts array?